docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,23 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
+* is taking CS2103/CS2103T in NUS
+* wants to keep track on his/her learning progress
+* is willing to have good team management
 * has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+* prefers desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* never miss LOs hidden in the nested collapsible list
+* keep track of your own progress on a week by week basis
+* keep track of your teammates' completion of TO-DO list(at most 20 tasks per day) and learning progress (providing proof for peer evaluation)
+* view the teammates' weekly plan in phase B (to know the upcoming updates even before they send any pull requests to github repo)
```

Comment:
As I said in the tutorial, it might overlap with the issue tracker functionality.